includes/ziti/ziti_log.h

```diff
@@ -91,6 +91,8 @@ ZITI_FUNC extern void ziti_log_set_level(int level);
 // don't use directly
 ZITI_FUNC extern int ziti_log_level();
 
+ZITI_FUNC extern int get_debug_level(char* log_level);
```

Comment:
`ziti_set_log_level(char* log_level)` sounds nice to me! :) the only time getting it is super useful is when deciding IF i want to log something.  if log_level > ERROR. so it might be useful to have a "getter" too?

Reply:
There is `ziti_log_level()` which does exactly that, although it is prefaced with a threat to "not use directly"

Reply:
I have removed the get_debug_level function and added ziti_log_set_level_by_label(const char *label). I also added ziti_log_level_label() function to return the string label value. I need the string value to save into the config file.